_posts/2020-07-11-Non-Binary.md

```diff
@@ -16,7 +16,7 @@ social-preview-image: /images/I'm Non-Binary 2 thumbnail.jpg
 That's the umbrella over my gender! This is a discovery that elates me! It brings me such joy, to know that this is how God made me. Ironic, isn't it? "But you're a programmer; you love binary!" 😝
 
 
-[Non-Binary means neither male nor female](https://en.wikipedia.org/wiki/Non-binary_gender), but it’s a very broad umbrella term &ndash; lots of Non-Binary people experience and express this in many different ways. For me, I simply don’t particularly care about my gender. I don’t care what pronouns people use for me, either. For a long time, I've let folks use whatever pronouns they want for me. As long as they show common respect and decency, it doesn't really matter to me. Of course, I take the time and care to show them the common respect and decency of knowing and honoring their genders and pronouns &ndash; that should go without saying. Everyone's gender is valid because it's an external expression of their internal self.
+[Non-Binary means neither male nor female](https://en.wikipedia.org/wiki/Non-binary_gender), but it’s a very broad umbrella term &ndash; lots of Non-Binary people experience and express this in many different ways. For me, I simply don’t particularly care about my gender. I don’t care what [pronouns](/Pronouns) people use for me, either. For a long time, I've let folks use whatever pronouns they want for me. As long as they show common respect and decency, it doesn't really matter to me. Of course, I take the time and care to show them the common respect and decency of knowing and honoring their genders and pronouns &ndash; that should go without saying. Everyone's gender is valid because it's an external expression of their internal self.
```

Comment:
Probably expand the linked text to "I don't care what pronouns people use for me"?